awscli/__init__.py

```diff
@@ -35,6 +35,7 @@
 
 
 EnvironmentVariables = {
+    'ca_bundle': ('ca_bundle', 'AWS_CA_BUNDLE', None, None),
```

Comment:
This is more a note for our team, but I think it would be worth moving this into botocore's `Session` so that python SDK users could also leverage setting a CA bundle in the `~/.aws/config` file and `AWS_CA_BUNDLE` env var.